src/Proto.Actor/Messages/Messages.cs

```diff
@@ -28,6 +28,14 @@ private Restarting()
         }
     }
 
+    public sealed partial class Touch : IAutoRespond
```

Comment:
New message, Touch. auto responds with Touched.
From filesystem touch, not kinky :)

Allows developers to do liveliness tests of actors, pass a touch and if you get a touched back, the actor is responding properly